### ISSUE TITLE
Quantizing Mobilenet

### ISSUE THREAD

**Issue by skvenka5** [eladhoffer/convNet.pytorch#15]
I am trying to quantize mobilenet model in the same how you have implemented resnet (https://github.com/eladhoffer/convNet.pytorch). To accomplish this I added the following lines in models/mobilenet .py

from .modules.quantize import QConv2d, QLinear, RangeBN
torch.nn.Linear = QLinear
torch.nn.Conv2d = QConv2d
torch.nn.BatchNorm2d = RangeBN

But, on training the loss is going to nan. It will be of great help if you could provide some inputs on this.

Thanks in advance

regards
Shreyas

**Comment by talenz** [comment]
Quantizing on ResNet-18 also goes NaN
